src/main/java/com/jcabi/beanstalk/maven/plugin/AbstractBeanstalkMojo.java

```diff
@@ -366,6 +348,11 @@ protected void validate(final ZipFile zip) throws MojoFailureException {
                 );
             }
         }
+        if (files < 1) {
+            throw new MojoFailureException(
+                ".ebextensions contains no config files."
+            );
+        }
```

Comment:
This block was the difference between the new version of this method and the original. The tests I moved over from WarFileTests required this behavior.